src/overlays/actors/ovl_En_Sw/z_en_sw.h

```diff
@@ -33,7 +33,7 @@ typedef struct EnSw {
     /* 0x0418 */ char unk_418[8];
     /* 0x0420 */ f32 rotateMag;
     /* 0x0424 */ char unk_424[0x8];
-    /* 0x042C */ u8 moveGoldBool; // set during EnSw_MoveGold, rever read.
+    /* 0x042C */ u8 goldMoveBool; // set during EnSw_GoldMove, rever read.
```

Comment:
```suggestion
    /* 0x042C */ u8 goldMoveBool; // set during EnSw_GoldMove, never read.
```

the last commit is never the last commit is never the last commit is never